lib/filter-stream.js

```diff
@@ -190,6 +191,18 @@ function FilterStream( blockMap, options ) {
 
   },
 
+  _flush( done ) {
+
+    var buffer = Buffer.concat( this._buffers )
+    var block = Buffer.alloc( this.blockSize )
+
+    buffer.copy( block )
```

Comment:
Also, is there a chance for `Buffer.concat( this._buffers )` to result in a buffer greater than `this.blockSize`?